onadata/libs/utils/chart_tools.py

```diff
@@ -435,6 +436,16 @@ def get_field_label(field, language_index=0):
     return field_label
 
 
+def _get_form_field_from_name_or_xpath(xform, name=None, xpath=None):
+    """
+    Given a name or xpath, get xform field
+    """
+    if name:
+        return get_field_from_field_name(name, xform)
+    elif xpath:
+        return get_field_from_field_xpath(xpath, xform)
```

Comment:
```suggestion
    return get_field_from_field_xpath(name or xpath, xform)
```

Reply:
I'm not sure if this would work, because we need to call a different method when we have `name` from when we have `xpath`

Reply:
Aah...cool. My bad,  I thought it was the same function